Ryujinx/Ui/SwitchSettings.cs

```diff
@@ -0,0 +1,431 @@
+                }
+            }
+            else { Button.SetStateFlags(0, true); }
+        }
+
+        private void AddDir_Pressed(object obj, EventArgs args)
+        {
+            if (Directory.Exists(_addGameDirBox.Buffer.Text)) { _gameDirsBoxStore.AppendValues(_addGameDirBox.Buffer.Text); }
+
+            _addDir.SetStateFlags(0, true);
+        }
+
+        private void BrowseDir_Pressed(object obj, EventArgs args)
+        {
+            FileChooserDialog fc = new FileChooserDialog("Choose the game directory to add to the list", this, FileChooserAction.SelectFolder, "Cancel", ResponseType.Cancel, "Add", ResponseType.Accept);
+
+            if (fc.Run() == (int)ResponseType.Accept)
+            {
+                _gameDirsBoxStore.AppendValues(fc.Filename);
+            }
+
+            fc.Destroy();
+
+            _browseDir.SetStateFlags(0, true);
+        }
+
+        private void RemoveDir_Pressed(object obj, EventArgs args)
+        {
+            TreeSelection selection = _gameDirsBox.Selection;
+
+            selection.GetSelected(out TreeIter treeiter);
+            _gameDirsBoxStore.Remove(ref treeiter);
+
+            _removeDir.SetStateFlags(0, true);
+        }
+
+        private void CustThemeToggle_Activated(object obj, EventArgs args)
+        {
+            if (_custThemeToggle.Active == false)
+            {
+                _custThemePath.Sensitive      = false;
+                _custThemePathLabel.Sensitive = false;
+                _browseThemePath.Sensitive    = false;
+            }
+            else
+            {
+                _custThemePath.Sensitive      = true;
+                _custThemePathLabel.Sensitive = true;
+                _browseThemePath.Sensitive    = true;
+            }
+        }
+
+        private void BrowseThemeDir_Pressed(object obj, EventArgs args)
+        {
+            FileChooserDialog fc = new FileChooserDialog("Choose the theme to load", this, FileChooserAction.Open, "Cancel", ResponseType.Cancel, "Select", ResponseType.Accept);
+            fc.Filter = new FileFilter();
+            fc.Filter.AddPattern("*.css");
+
+            if (fc.Run() == (int)ResponseType.Accept)
+            {
+                _custThemePath.Buffer.Text = fc.Filename;
+            }
+
+            fc.Destroy();
+
+            _browseThemePath.SetStateFlags(0, true);
+        }
+
+        private void SaveToggle_Activated(object obj, EventArgs args)
+        {
+            List<string> gameDirs = new List<string>();
+
+            _gameDirsBoxStore.GetIterFirst(out TreeIter iter);
+            for (int i = 0; i < _gameDirsBoxStore.IterNChildren(); i++)
+            {
+                _gameDirsBoxStore.GetValue(iter, i);
+
+                gameDirs.Add((string)_gameDirsBoxStore.GetValue(iter, 0));
+
+                _gameDirsBoxStore.IterNext(ref iter);
+            }
+
+            if (_errorLogToggle.Active)        SwitchConfig.LoggingEnableError        = true;
```

Comment:
```suggestion
            SwitchConfig.LoggingEnableError        = _errorLogToggle.Active;
```
and same for other lines.